test/rules/always_put_control_body_on_new_line.dart

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// test w/ `pub run test -N always_put_control_body_on_new_line`
+
+testIfElse() {
```

Comment:
Reading the [Flutter rule](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#separate-the-if-expression-from-its-statement) I think it should.